core/src/main/java/org/mskcc/cbio/portal/model/AlleleSpecificCopyNumber.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2019 Memorial Sloan-Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
+ * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
+ * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
+ * obligations to provide maintenance, support, updates, enhancements or
+ * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
+ * liable to any party for direct, indirect, special, incidental or
+ * consequential damages, including lost profits, arising out of the use of this
+ * software and its documentation, even if Memorial Sloan-Kettering Cancer
+ * Center has been advised of the possibility of such damage.
+ */
+
+/*
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package org.mskcc.cbio.portal.model;
+
+import java.io.Serializable;
+import java.util.*;
+
+/**
+ *
+ * @author averyniceday.
+ */
+public class AlleleSpecificCopyNumber implements Serializable {
+
+    private long mutationEventId;
+    private int geneticProfileId;
+    private int sampleId;
+    private Integer ascnIntegerCopyNumber;
+    private String ascnMethod;
+    private Float ccfMCopiesUpper;
+    private Float ccfMCopies;
+    private Boolean clonal;
+    private Integer minorCopyNumber;
+    private Integer mutantCopies;
+    private Integer totalCopyNumber;
+
+    public AlleleSpecificCopyNumber(Map<String,String> ascnData) {
```

Comment:
just linking to the comment above about the constructor having to be changed -- these are all going to return null because the keys are being lower cased before being placed in the map

Reply:
👍